alpine/2.3.6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Only alpine version that suports rethinkdb is edge
+FROM alpine:edge
+
+ENV RETHINKDB_VERSION 2.3.6-r1
+
+RUN apk add --no-cache \
+    rethinkdb=$RETHINKDB_VERSION
+
+VOLUME ["/data"]
+
+WORKDIR /data
+
+EXPOSE 28015 29015 8080
```

Comment:
I'd prefer it if the `EXPOSE` directive were moved to come after `CMD`, with the comment stating which ports are which, [the way it is in the other Dockerfiles](https://github.com/rethinkdb/rethinkdb-dockerfiles/blob/master/jessie/2.3.6/Dockerfile#L20-L23).